upi/azure/azuredeploy.json

```diff
@@ -0,0 +1,1072 @@
+{
+  "$schema" : "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion" : "1.0.0.0",
+  "parameters" : {
+    "image" : {
+      "type" : "string",
+      "metadata" : {
+        "description" : "OS to use."
+      }
+    },
+    "BootstrapIgnition" : {
+      "type" : "string",
+      "minLength" : 1,
+      "metadata" : {
+        "description" : "User name for the Virtual Machine and OpenShift Webconsole."
```

Comment:
the description needs to be updated.

Reply:
These are all related to the first commit and changed in the second one. @abhinavdahiya should I just go ahead and squash them into a single commit with 2 authors?

Reply:
This is fixed in the second commit.